### DISCUSSION TITLE
Possible error on the documentation for the skipWhile operator

### DISCUSSION THREAD

**Discussion by R1Daneel** [ReactiveX/rxjs#6661] [Report issues other than bug]
The documentation for the operator skipWhile says:

> Skips all the notifications with a truthy predicate. It will not skip the notifications when the predicate is falsy.  It can also be skipped using index. Once the predicate is true, it will not be called again.

The error is the sentence `Once the predicate is true, it will not be called again.` in that the predicate is no longer called when it becomes for the first time falsy and not truty.

**Comment by cartant** [comment]
Yeah, you are correct; that is an error. If you're able to open a PR to fix it, I'll happily merge it. Thanks.

**Reply by R1Daneel** [reply]
Yes, I will happy to do that
